specification/mediaservices/resource-manager/Microsoft.Media/preview/2020-02-01-preview/MediaGraphs.json

```diff
@@ -0,0 +1,901 @@
+      "post": {
+        "summary": "Start a Media Graph",
+        "description": "Start a Media Graph in the Media Services account.",
+        "operationId": "MediaGraphs_Start",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Start Media Graph": {
+            "$ref": "examples/media-graph-start.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/stop": {
+      "post": {
+        "summary": "Stop a Media Graph",
+        "description": "Stop a Media Graph in the Media Services account.",
+        "operationId": "MediaGraphs_Stop",
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/MediaServicesAccountName"
+          },
+          {
+            "name": "mediaGraphName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Media Graph name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Stop Media Graph": {
+            "$ref": "examples/media-graph-stop.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaServices/{accountName}/mediaGraphs/{mediaGraphName}/operationsStatus/{operationId}": {

```

Comment:
>operationsStatus [](start = 155, length = 16)

What's the difference between `operationResults` and `operationsStatus` resource type?

Reply:
operationResults is for location header, and operationStatus is for azure-asyncoperation header.

Reply:
Do you need both?

Reply:
We are planning to only use location header for the public preview. For this change, since the previous private preview version uses both, to make the change set minimum, we decided to keep it that way.